src/07-even-more-loops.js

```diff
@@ -28,7 +36,17 @@ function shoutForLoop(array) {}
  *  shoutWhileLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutWhileLoop(array) {}
+function shoutWhileLoop(array) {
```

Comment:
Probably not most intuitive most people to write this loop to iterate backward... but it will work so 🤷🏾‍♀️